src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx

```diff
@@ -73,6 +80,15 @@ function LoggedInNav({ queryRef }: Props) {
       <NavElement>
         <Dropdown mainText={query.viewer.user.username} shouldCloseOnMenuItemClick>
           <TextButton text="My Gallery" onClick={() => push(`/${username}`)} />
+          {isFeatureEnabled(FeatureFlag.POSTER_MINT) && (
```

Comment:
it looks like we want to left align the dropdown items now that we have the notification dot so that the dot doesn't displace text
![Screen Shot 2022-06-01 at 20 49 44](https://user-images.githubusercontent.com/80802871/171397979-846021e6-8fd5-48db-8331-e06f7db85890.png)

vs
![Screen Shot 2022-06-01 at 20 49 47](https://user-images.githubusercontent.com/80802871/171397898-105456fd-e488-4ddc-a7df-de2de1c13c5c.png)


but now that I think about it, having them right aligned is kinda convenient for clicking the username -> clicking a dropdown option, so I've asked jess in Figma. will keep you posted - but either way we'll want to make sure the notification dot doesn't make the options unaligned.

Reply:
Got it. Updated to right align for now but wth the dot in the right side. Open for changes after the discussion